sdk/cosmos/azure_data_cosmos/src/location_cache.rs

```diff
@@ -0,0 +1,570 @@
+use std::{collections::HashMap, sync::Mutex, time::SystemTime};
```

Comment:
You can put `#![allow(dead_code)]` at the top of this file for now. The build is complaining that nothing uses `LocationCache` yet, but that's expected.